app/flatpak-builtins-build-init.c

```diff
@@ -292,8 +293,15 @@ flatpak_builtin_build_init (int argc, char **argv, GCancellable *cancellable, GE
         return FALSE;
     }
 
+  g_autoptr(FlatpakRef) ref = flatpak_ref_parse (sdk_ref, NULL);
+  const char *sdk_branch = flatpak_ref_get_branch (ref);
+
+  if (sdk_branch == NULL) {
+    sdk_branch = default_branch;
```

Comment:
I don't see how this could ever happen. We already used default_branch when  creating the sdk_ref, and if we did create a ref it will have a branch (or it would have failed)

Reply:
I don't know the code so I trust you :) I was not sure if the ref here could be an unqualified one. I'll remove the if.